File: consys-core/src/main/scala/de/tuda/stg/consys/core/store/cassandra/CassandraConsistencyLevels.scala
package de.tuda.stg.consys.core.store.cassandra

import de.tuda.stg.consys.core.store.StoreConsistencyLevel
import de.tuda.stg.consys.core.store.cassandra.levels.{Strong, Weak}

/**
 * Created on 27.01.20.
 *
 * @author Mirko Köhler
 */
object CassandraConsistencyLevels {
	val WEAK : StoreConsistencyLevel = Weak
	val STRONG : StoreConsistencyLevel = Strong
}

File: consys-japi/src/main/java/de/tuda/stg/consys/japi/binding/cassandra/CassandraConsistencyLevels.java
package de.tuda.stg.consys.japi.binding.cassandra;

import de.tuda.stg.consys.core.store.ConsistencyLevel;
import de.tuda.stg.consys.core.store.cassandra.CassandraStore;
import de.tuda.stg.consys.core.store.cassandra.levels.Mixed$;
import de.tuda.stg.consys.core.store.cassandra.levels.Strong$;
import de.tuda.stg.consys.core.store.cassandra.levels.Weak$;

public class CassandraConsistencyLevels {
    public static final ConsistencyLevel<CassandraStore> STRONG = Strong$.MODULE$;
    public static final ConsistencyLevel<CassandraStore> WEAK = Weak$.MODULE$;
    public static final ConsistencyLevel<CassandraStore> MIXED = Mixed$.MODULE$;
}
